samples/kotlin_compose/src/main/java/io/customer/android/sample/kotlin_compose/util/Logger.kt

```diff
@@ -0,0 +1,29 @@
+package io.customer.android.sample.kotlin_compose.util
+
+import android.util.Log
+
+class Logger {
+    companion object {
+        private const val TAG = "KOTLIN_COMPOSE"
+    }
+
+    fun d(message: String) {
+        Log.d(TAG, message)
+    }
+
+    fun v(message: String) {
+        v(message, null)
+    }
+
+    fun v(message: String, throwable: Throwable?) {
+        Log.v(TAG, message, throwable)
+    }
```

Comment:
We could use default parameters here 🤔 

```suggestion
    fun v(message: String, throwable: Throwable? = null) {
        Log.v(TAG, message, throwable)
    }
```